CMakeLists.txt

```diff
@@ -77,6 +77,17 @@ if (USE_AVX2)
 	endif()
 endif(USE_AVX2)
 
+option(USE_ARM "Compile for ARM compatible microarchitectures" ${PIONEER_TARGET_ARM})
+option(PIONEER_TARGET_RASPBERRY_PI "Compile for Raspberry Pi specifically (degrades to OpenGL 3.1)" OFF)
+
+if (USE_ARM)
```

Comment:
Note that we now have autodetection for ARM platforms based on the `CMAKE_SYSTEM_PROCESSOR` variable - compilation for ARM64 should set `CMAKE_SYSTEM_PROCESSOR` to one of `aarch64` or `ARM64` (likely `aarch64` since it's a linux target OS - this should be done by whatever external tooling or option sets up the correct compiler toolchain for Raspberry Pi).

...unless "normal" Raspberry Pi uses standard GCC builds in which case this should be set by the `PIONEER_TARGET_RASPBERRY_PI` option before we `include(cmake/TargetArchitecture.cmake)`. I'm used to working with the RPi Pico which has a separate GCC toolchain from the system compiler.